components/Moola.vue

```diff
@@ -103,13 +103,18 @@
                 return split.join('.');
             },
 
-            checkDecimals(event) {
+            blur(event) {
                 var val = this.removeCommas(event.target.value);
```

Comment:
This should probably be renamed to `removeDelimiters`

Reply:
Changed!